### DISCUSSION TITLE
Autofill GraphQL request from Documentation Fields

### DISCUSSION THREAD

**Discussion by jostFT** [Kong/insomnia#3610] [Ideas]
**Is your feature request related to a problem? Please describe.**
Whenever I have to setup a new GraphQL request, and the request is quite hefty (say, 5-6 parameters and/or a ton of return data). It is incredibly tedious to manually type it in. Especially, when I want to structure it to support Query Variables.
 
**Describe the solution you'd like**
I would like the ability to press an "import" button within the documentation section that would then populate the entire request into the GraphQL window. Granted, leaving out the actual input values. 

**Describe alternatives you've considered**
Alternatively, make the text within the documentation area highlightable, so I can grab a hold of the the query and paste it in myself. 

**Additional context**
Here is an example of what I'm thinking:
![Screen Shot 2020-11-06 at 11 50 43](https://user-images.githubusercontent.com/46314409/98363505-fb2fec00-2026-11eb-9b86-92c9b12d5b0c.png)

I am running **Insomnia Core 2020.4.2**
on **Mac OS Catelina 10.15.6**

**Comment by jostFT** [comment]
bump

**Comment by ehenson** [comment]
This will be a very handy feature.  I'm having to use other tools to perform this and if Insomnia could have this feature then I can exclusively use this application for all of my projects.
